main/src/org/destinationsol/menu/InputMapScreen.java

```diff
@@ -37,102 +38,105 @@
         float contentW = .8f;
-        float col0 = r / 2 - contentW / 2;
-        float row0 = .2f;
-        float row = row0;
-        float bgGap = MenuLayout.BG_BORDER;
+        float col0 = resolutionRatio / 2 - contentW / 2;
+        float row = 0.2f;
         float bigGap = SMALL_GAP * 6;
         float headerH = .03f;
 
-
-        // list header & controls
-        listHeaderPos = new Vector2(col0 + HEADER_TEXT_OFFS, row + HEADER_TEXT_OFFS); // offset hack
+        // List header & controls
+        listHeaderPos = new Vector2(col0 + HEADER_TEXT_OFFSET, row + HEADER_TEXT_OFFSET); // offset hack
         float listCtrlW = contentW * .15f;
         Rectangle nextArea = new Rectangle(col0 + contentW - listCtrlW, row, listCtrlW, headerH);
-        nextCtrl = new SolUiControl(nextArea, true, gameOptions.getKeyRight());
-        nextCtrl.setDisplayName(">");
-        controls.add(nextCtrl);
+        nextControl = new SolUiControl(nextArea, true, gameOptions.getKeyRight());
+        nextControl.setDisplayName(">");
+        controls.add(nextControl);
         Rectangle prevArea = new Rectangle(nextArea.x - SMALL_GAP - listCtrlW, row, listCtrlW, headerH);
-        prevCtrl = new SolUiControl(prevArea, true, gameOptions.getKeyLeft());
-        prevCtrl.setDisplayName("<");
-        controls.add(prevCtrl);
+        previousControl = new SolUiControl(prevArea, true, gameOptions.getKeyLeft());
+        previousControl.setDisplayName("<");
+        controls.add(previousControl);
         row += headerH + SMALL_GAP;
 
-        // list
+        // List
         float itemRowH = .04f;
         float listRow0 = row;
-        itemCtrls = new SolUiControl[Const.ITEM_GROUPS_PER_PAGE];
+        itemControls = new SolUiControl[Const.ITEM_GROUPS_PER_PAGE];
         for (int i = 0; i < Const.ITEM_GROUPS_PER_PAGE; i++) {
             Rectangle itemR = new Rectangle(col0, row, contentW, itemRowH);
             SolUiControl itemCtrl = new SolUiControl(itemR, true);
-            itemCtrls[i] = itemCtrl;
+            itemControls[i] = itemCtrl;
             controls.add(itemCtrl);
             row += itemRowH + SMALL_GAP;
         }
         listArea = new Rectangle(col0, row, contentW, row - SMALL_GAP - listRow0);
         row += bigGap;
 
-        // detail header & area
+        // Detail header & area
         row += headerH + SMALL_GAP;
         float itemCtrlAreaW = contentW * .4f;
-        itemCtrlArea = new Rectangle(col0 + contentW - itemCtrlAreaW, row, itemCtrlAreaW, .2f);
-        detailArea = new Rectangle(col0, row, contentW - itemCtrlAreaW - SMALL_GAP, itemCtrlArea.height);
-        row += detailArea.height;
+        itemControlsArea = new Rectangle(col0 + contentW - itemCtrlAreaW, row, itemCtrlAreaW, .2f);
+        detailsArea = new Rectangle(col0, row, contentW - itemCtrlAreaW - SMALL_GAP, itemControlsArea.height);
+        // row += detailsArea.height;
```

Comment:
Because, row is not used anymore after this. So, checkstyle gives a warning about statement being useless. With that being said, I did not want to remove this, in case we wanted to add something else to this later.. hence the commenting out.